app.py

```diff
@@ -174,13 +208,28 @@ def prepare_metadata():
     sort_image_captions()
 
 
-# Deletes all files uploaded through the GUI and removes them from the dict
-def clean_up():
+# Deletes files uploaded through the GUI and removes them from the dict
+# If 'self' is given then only the current user's images are deleted
+def clean_up_user_images(self):
```

Comment:
This is really weird to me. I've never seen self used this way. Why not just use an argument that defaults to false?

```
def clean_up_user_images(user=None):
    img_prefix = get_user_img_prefix(user) if user else temp_img_prefix
```

Reply:
after reading through my code a bit I realized that using `self` this way is a remnant of my process to get this feature working, I'll update it so the original handler grabs the cookie out of `self` and passes that instead, in which case I'll be passing the cookie around in the same way. That'll be more proper python code. (I think I originally grabbed more than just the cookie from `self` at one point)